lib/triangle_farms/calendar/event_api/client.ex

```diff
@@ -13,8 +13,10 @@ defmodule TriangleFarms.Calendar.EventAPI.Client do
   def authorize do
     with %{body: body} <- HTTPoison.post!(@aud, token_params()) do
       case Poison.decode!(body) do
-        %{"access_token" => token} -> {:ok, %Client{token: token}}
-        %{"error_description" => msg} -> {:error, msg}
+        %{"access_token" => token} ->
```

Comment:
It's just a personal preference of mine to have the return values of cases on their own lines. I didn't actually intend to push this. :|

Reply:
Looking at it, I like that better as well :)